packages/catalog-search/src/styles/_CurrentRefinements.scss

```diff
@@ -11,3 +11,9 @@
 .fe__current-refinement-button--inverse {
   color: $white;
 }
+
+.justify-center {
```

Comment:
nit: would recommend either putting this class name definition under another class name specific to the component, or do something similar to the other class names in this file (e.g., prefix like the above `.fe__current-refinements-*` does).